### ISSUE TITLE
Matlab extension crashes Git source control

### ISSUE THREAD

**Issue by Glide-RohanRupela** [Gimly/vscode-matlab#178]
Type: <b>Bug</b>

Installed the Matlab Extension by Xavier Hahn. 
Sometime after switching the language setting to MATLAB, the app throws an error.
Due to that VS is not able to identify Git repository.
As a result we are not able to track all the changes being done in the repo through vs code.

Extension version: 2.3.1
VS Code version: Code 1.76.1 (5e805b79fcb6ba4c2d23712967df89a089da575b, 2023-03-08T16:32:00.131Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz (8 x 2419)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|7.74GB (1.56GB free)|
|Process Argv|--crash-reporter-id a383d3f3-abdc-4814-827f-79f560bb8b14|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
pynewvextcfv2:30681851
azure-dev_surveyone:30548225
vscccc:30610679
pyindex848:30662994
nodejswelcome1cf:30587006
2e4cg342:30602488
pyind779:30671433
f6dab269:30613381
pythonsymbol12:30671437
showlangstatbar:30672706
vsccsb:30677849
pythonb192:30669360
funwalk2:30682974
pythonms35:30680984

```

</details>

**Comment by zm-cttae** [comment]
@Gimly will need to do `npm i npm-run-all --save-dev`, bump version & publish #174 to fix this

**Comment by Gimly** [comment]
Should be fixed in v3.0.1
